app/api/schema/settings.py

```diff
@@ -129,6 +129,13 @@ class Meta:
     gs_secret = fields.Str(allow_none=True)
     gs_bucket_name = fields.Str(allow_none=True)
 
+    #
+    # Google reCAPTCHA
+    #
+
+    captcha_site_key = fields.Str(allow_none=True)
```

Comment:
Add a boolean field - google_captcha_enabled

Reply:
Doing the following

Reply:
@iamareebjamal Please Review now